data/data/alibabacloud/cluster/outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "vpc_id" {
+  value = module.vpc.vpc_id
+}
+
+output "vswitch_ids" {
```

Comment:
The outputs from this stage or part of the installer will be used as inputs to the next stage which is the bootstrap. There are two variables the bootstrap machine requires which should exist here in order for the bootstrap machine to succeed. 
Add these two fields:
- vswitch_id
- slb_id

View this file for reference: https://github.com/openshift/installer/pull/5018/files#diff-7f5a243bb3f02a465fc5fbd4332d35f238fc84c6fe494220e6582eaccc2aa38b

Reply:
Okay, I understand the problem here. Bootstrap stage variables need to be obtained in the outputs of the cluster stage, so I unified the outputs and variables on both sides, and cleaned up some unnecessary items。commit: https://github.com/openshift/installer/pull/5018/commits/dc4d5c8659a3d8769fa2c7511c7f13add99aa199